### ISSUE TITLE
Council / Fey Armor overrides flight from Creative Flight Pylon

### ISSUE THREAD

**Issue by Creepercharge** [Mithion/Mana-And-Artifice#911]
**Describe the bug**
Council and Fey Armor Levitation passives override the flight effect of the creative flight pylon 

**To Reproduce**
1. Set up a creative flight pylon
2. Equip either Council or Fey armor sets
3. Fly with and without the set bonuses active

**Expected behavior**
Creative flight from the pylon should allow for faster flight within the effect radius without having to unequip the Council or Fey faction armor sets

**Version Information (please complete the following information):**
 - Mod Version: 3.0.0.22
 - Forge Version 47.2.20

**Comment by CFlatFlat** [comment]
It overrides any kind of creative-esque flight from other mods as well, which is a real pain.

**Comment by Creepercharge** [comment]
Yeah maybe not outright granting creative flight since that would render the pylon version useless
